json/navigation-conf/us.json

```diff
@@ -59,8 +59,8 @@
       "sections": [
         {
           "title": "Soccer",
-          "path": "football",
-          "mobileOverride": "section-list"
+          "path": "us/soccer",
+          "mobileOverride": "section-front"
```

Comment:
This `mobileOverride` controls the type of the page (e.g. tag list / front / section list) this nav item goes to.   

For reference, if we do not set the `mobileOverride`, MAPI will attempt to check whether the path is a front name and set it as a front page.  Otherwise MAPI will generate a link for tag list.

Details can be found [here](https://github.com/guardian/mobile-apps-api/blob/main/mobile-fronts/app/data/navigation/FrontsLayoutLoader.scala#L30-L44)